interface/forms/vitals/templates/vitals/vitals_textbox_conversion.tpl

```diff
@@ -0,0 +1,74 @@
+<!-- USA row comes first -->
+    {if $units_of_measurement == $MEASUREMENT_METRIC_ONLY}
+    <tr class="hide">
+    {else}
+    <tr>
+    {/if}
+    {if $units_of_measurement == $MEASUREMENT_PERSIST_IN_METRIC}
+        <td class="unfocus graph" id="{$input}">
+    {else}
+        <td class="graph" id="{$input}">
+    {/if}
+            {xlt t=$title}
+        </td>
+    {if $units_of_measurement == $MEASUREMENT_PERSIST_IN_METRIC}
+        <td class="unfocus">
+    {else}
+        <td>
+    {/if}
+            {xlt t=$unit}
+        </td>
+    {if $units_of_measurement == $MEASUREMENT_PERSIST_IN_METRIC}
+        <td class="valuesunfocus">
+    {else}
+        <td class='currentvalues p-2'>
+    {/if}
+            <input type="text" class="form-control" size='5' name='{$input}' id='{$input}_input'
```

Comment:
2 attr's above and in other places in this PR for the {$input}

Reply:
Should be fixed.